script/setup/install-runc

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# Builds and installs runc to /usr/local/go/bin based off
+# the commit defined in RUNC.md
```

Comment:
There's a line in `.travis.yml` which installs runc, it should probably be replaced with `./script/setup/install-runc`